runOnPc/make_stub_files.py

```diff
@@ -1629,10 +1629,13 @@ def make_stub_file(self, fn):
         # out_fn = os.path.join(self.output_directory, base_fn)
         # out_fn = out_fn[:-3] + '.pyi'
         out_fn = fn + 'i'
-        self.output_fn = os.path.normpath(out_fn)
```

Comment:
Removing this line actually seems to break the `make_stub_files.py` (at least in linux anyways). Looks like it doesn't actually write the file without it.

Reply:
Yes you're right. This is a mistake I made when submitting the PR> I will fix and update